src/main/kotlin/frc/robot/subsystems/conveyor/ConveyorIOSim.kt

```diff
@@ -0,0 +1,47 @@
+package frc.robot.subsystems.conveyor
+
+import com.revrobotics.CANSparkBase
+import com.revrobotics.CANSparkMax
+import edu.wpi.first.math.controller.PIDController
+import edu.wpi.first.math.controller.SimpleMotorFeedforward
+import edu.wpi.first.units.Angle
+import edu.wpi.first.units.MutableMeasure
+import edu.wpi.first.units.Units
+import edu.wpi.first.units.Velocity
+import edu.wpi.first.wpilibj.Timer
+import frc.robot.lib.motors.SparkMaxSim
+
+class ConveyorIOSim : ConveyorIO {
+    override val inputs = LoggedConveyorInputs()
+    private val conveyor = SparkMaxSim(
```

Comment:
If the real system uses a TalonFX, so should the simulated system.

Reply:
685dc4c001ae7927cf4a547d85bbfc87cc75fe50